app/views/friendships/_friends_list.html.erb

```diff
@@ -0,0 +1,39 @@
+<% obj.each do |friendship| %>
+<tbody class = "friend-row">
+	<tr>
+		<td ><strong>Summoner Name: <%= friendship.friend.name.capitalize %></strong></td>
+		<% if @friends_list_code == 'pending' %> 
+		  <td ><strong> <%if friendship.initiator == false %>
+		                  Received: 
+		                <% else %>
+		                  Sent: 
+		                <% end %> 
+		    <%= "#{time_ago_in_words(friendship.created_at)}".capitalize %> ago</strong></td>
+		<% else %>
+		  <td ><strong>Friends Since: <%= "#{time_ago_in_words(friendship.request_responded_at)}".capitalize %> ago</strong></td>
+		<% end %>
+	  <td class="champ-img" rowspan="3" scope="rowgroup"><span class="imgWrap"><%= link_to image_tag("icons/AhriSquare.png"), profile_path(friendship.friend.id)  %></span>
+    </td>
+	</tr>
+	<tr>
+		<td ><strong>Friendship Status: <%= friendship.friendship_status.capitalize %></strong></td>
+	  <td ><strong>Member Since: <%= "#{time_ago_in_words(friendship.friend.created_at)}".capitalize %> ago</strong></td>
+	</tr>
+	<tr data-no-turbolink>
+	  <% if @friends_list_code == 'pending' %> 
+	    <%if friendship.initiator == false %>
+		    <td ><%= link_to  "Accept Request", decide_friendships_path(friend_email: current_user.email, requester_email: User.find(friendship.friend.id).email, request_token: 'from-friends-list', response_code: 'accept'), :class =>"btn btn-success" %> </td>
```

Comment:
You really shouldn't be using hash rocket syntax anywhere.  That is old syntax.  This can be made into `class: "btn btn-success"`